src/test/resources/test_scripts/stages-with-wait.groovy

```diff
@@ -1,7 +1,7 @@
 node {
    stage 'Stage 1'
-   sh 'sleep 3; echo `date` Stage 1;'
-   sh 'sleep 3; echo `date` Stage 1;'
+   sh 'sleep 6; echo `date` Stage 1a we need some time here for slower ath hosts;'
```

Comment:
sadly input will not do here, since we are testing the running state and the karaoke but I agree that this sleep stuff is evil.